### ISSUE TITLE
SIP REGISTRATION - IP BASED

### ISSUE THREAD

**Issue by xceedconnections** [magnussolution/magnusbilling7#533]
HEllo,

i added a sip user into vicidial with ip based registration and its working fine with vicidial default sip port 5060.

i have 1 more VICIDIAL dialer with sip port changed from 5060 to 7777.
we are unable to make calls, 

can some one guide please.

**Comment by ajtel** [comment]
did you OPEN PORTS IN FIREWALL?

**Comment by xceedconnections** [comment]
how to open it.

vicidial sip port is 7777.
magnus sip port is 5060

in vicidial username password based registration is working fine.
but when using ip based, it not registered.

how to add in firewall?

**Comment by ajtel** [comment]
open in firewall of both apliances to allow traffic  (firewalld is normal firewall for both)

**Comment by xceedconnections** [comment]
is it necessary for both systems have same port open??

**Comment by ajtel** [comment]
yes

**Comment by magnussolution** [comment]
> is it necessary for both systems have same port open??
> […](#)

no, you can use on the client any port, and is not necessary open the port on MangusBilling. But if is IP authentication, you will set the insecure to INVITE, to asterisk not check the port

**Comment by Shahedsoft** [comment]
**Webrtc System User Dashbord**
When the customer login user dashboard can call with full system webphone webrtc

**Magnusbilling Customer Protal Customize**
Simple and easy used for customer portal
change the password call history signing up logging at user mode balance transfer System

**Softphone Customize Magnusbilling**
We customize Android and iOS webrtc phone

**Price For Both Platfrom**

**Magnusbilling android Softbilling**
Basic functionality available

**Contact us WhatsApp number**
Sale@ringme.us
+8801816676486
www.ringme.us
